internal/commands/login/inputs.go

```diff
@@ -30,23 +30,39 @@ type inputs struct {
 	PrivateAPIKey string
 	Username      string
 	Password      string
+	Browser       bool
 }
 
 func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
 	u := profile.Credentials()
 
-	if u == (user.Credentials{}) {
+	if i.Browser && (i.Username != "" || i.Password != "" || i.PublicAPIKey != "" || i.PrivateAPIKey != "") {
+		return errors.New("credentials will not be authenticated while using browser flag, please login with one or the other")
+	}
+
+	if u == (user.Credentials{}) || i.Browser {
 		if err := ui.OpenBrowser(apiKeysPage); err != nil {
-			ui.Print(terminal.NewErrorLog(errors.New("there was an issue opening your browser")))
+			ui.Print(terminal.NewWarningLog("there was an issue opening your browser"))
 		}
 	}
 
 	var questions []*survey.Question
 
-	switch i.AuthType {
-	case authTypeCloud:
+	switch {
+	case i.Browser:
+		questions = []*survey.Question{
+			{
+				Name:   inputFieldPublicAPIKey,
+				Prompt: &survey.Input{Message: "Public API Key"},
```

Comment:
I intentionally left it out since we're assuming the purpose of the browser is to **create new keys**, in which case, we do not want to populate it with a default key from a profile.

Reply:
I'd agree that's the purpose of the browser in the scenario where it organically happens, however if we're forcing the browser to launch and if the `--profile` set happens to already exist, providing this as a default value at least gives the user an idea of what they're overriding (should they use a new key).

i mostly asked this b/c i'm fairly certain that was the behavior you had coded before when we had a bunch of `|| i.Browser` conditions.  considering it's just a default value/display, i don't think it _really_ matters either way, the chances of the user being able to re-type the same private api key for the existing profile is **very slim**

i imagine regardless of the default value, we'll still be displaying the prompt/confirm to verify the user has an existing session and they need to confirm if they want to log that existing user out?

Reply:
I'm afraid that if it's provided as a default, the user will be inclined to just press enter thus defeating the prompt for the new keys created in the browser.

side note:  this is what I had before with the bunch of `|| i.Browser` conditions 🙃 
```
var defaultInput string
if !i.ForceBrowser {
  defaultInput = u.PublicAPIKey
}
```

Reply:
🤝 
will add the confirmation.